test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] [required] I would advise to remove this file because it is not related to your project. That will make your codes much more clean, readable, debuggable, and maintainable.

Reply:
Done